css_injector.admin.inc

```diff
@@ -133,13 +133,13 @@ function css_injector_edit($form, $form_state, $crid = NULL) {
     // Adding ace syntex highliter.
```

Comment:
Maybe also fix the typo while at it? (synt**e**x should be synt**a**x)

Reply:
It's spelled that way in several other places. :laughing: 

Time to fix it?

Reply:
I'm not against it ;-) Indeed it would be nice as there is at least one place in the user interface, but I guess it's not critical. 😄

Reply:
Already committed. :wink:

Reply:
Thanks @indigoxela and @yorkshire-pudding 🙏🏼 ...looks good 👍🏼